src/actions.js

```diff
@@ -534,6 +534,10 @@ export const fetchUserAuthMapping = async (dispatch) => {
   if (authzMappingURL === wtsAggregateAuthzPath) {
     authMapping = fetchedAuthMapping[hostnameWithSubdomain];
     aggregateAuthMappings = fetchedAuthMapping;
+    if (Object.keys(aggregateAuthMappings).length === 0) {
+      authMapping = fetchedAuthMapping;
+      aggregateAuthMappings[hostnameWithSubdomain] = fetchedAuthMapping;
```

Comment:
can you explain why this works? we are already doing `aggregateAuthMappings = fetchedAuthMapping;` in L536 so if L537 is true, doesn't that means `fetchedAuthMapping` is also empty?

Reply:
I think the fall back should happen in an much eariler place, like https://github.com/uc-cdis/data-portal/pull/1106/files#diff-98405f0e77bce0c636c493239fdc977dd78f320f5313d5378b701c3effef541cR518

Logic should be:
1. if `discoveryUseAggWTS` is on, try to fetch agg WTS path first. If that request fails with a non-200, then fetch local arborist
2. if `discoveryUseAggWTS` is off, then just fetch local arborist

Reply:
Yeah, you're right. All we're trying to do here is if agg WTS is on, and that `aggregateAuthMappings` is empty because there are no externally connected commons, return local arborist

Reply:
I'll see if I can move this to earlier

Reply:
Sure, Sai, go for it

Reply:
> if discoveryUseAggWTS is on, try to fetch agg WTS path first. If that request fails with a non-200, then fetch local arborist

One slight modification to this comment is that -- The wts/aggregate wouldn't respond with a non-200 response, but simply returns an empty object in this case. 

I've made necessary changes and pushed a fresh [commit](https://github.com/uc-cdis/data-portal/pull/1106/commits/e2f6543c04550599ed29f599485b26854ea67da2) to this branch, please take a look. 

`Note: This is agreed to be a band-aid solution until an efficient solution is introduced as a part of this ->` [JIRA ticket](https://ctds-planx.atlassian.net/browse/BRH-260)

Reply:
What about when WTS is unstable and returns 502? 🤔

Reply:
Ah! that is a good point. Co-incidentally, I think [this part](https://github.com/uc-cdis/data-portal/blob/e2f6543c04550599ed29f599485b26854ea67da2/src/actions.js#L511-L514) of the logic covers that case as well.

Since we get a `{}` response in either of the cases, we'd go ahead and pull data from local arborist